admin/bin/qserv-data-loader.py

```diff
@@ -146,8 +144,10 @@ def __init__(self):
                            help='Disable CSS updates.')
 
         group = parser.add_argument_group('Database options', 'Options for database connection')
-        group.add_argument('-H', '--host', dest='mysqlHost', default='localhost', metavar='HOST',
-                           help='Host name for czar mysql server, def: %(default)s.')
+        group.add_argument('-H', '--host', dest='czarHost', default='localhost', metavar='HOST',
+                           help='Host name for czar wmgr service, def: %(default)s.')
```

Comment:
In some sense czar is also a worker as it does some work :) I don't want to do renaming at this point, too many things to change. Maybe when we have nothing else to do. For now people just need to remember that wmgr also manages czar. Or think of wmgr as "world manager" :)